easybuild/easyconfigs/s/SciANN/SciANN-0.7.0.1-foss-2022a.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonBundle'
+
+name = 'SciANN'
+version = '0.7.0.1'
+
+homepage = 'https://github.com/ehsanhaghighat/sciann'
+description = """
+A Keras/Tensorflow wrapper for scientific computations and physics-informed deep learning
+using artificial neural networks.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('h5py', '3.7.0'),
+    ('scikit-learn', '1.1.2'),
+    ('TensorFlow', '2.11.0'),
+    ('pymatgen', '2023.3.10'),  # for pybtex and latexcodec
+    ('matplotlib', '3.5.2'),
+    ('pygraphviz', '1.10'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pydot', '2.0.0', {
```

Comment:
There is `pydot` as its own easyconfig: `pydot-1.4.2-GCCcore-11.3.0.eb`. Can you use that one as dependency instead of adding an extension?